src/main.js

```diff
@@ -107,17 +130,41 @@ export class Draggable {
         this._mouseup = (e) => {
             unbind(this.document, "mousemove", this._mousemove);
             unbind(this.document, "mouseup", this._mouseup);
+
+            if (this.clickMoveClick){
+                unbind(this.document, "keydown", this._keydown);
+                unbind(this.document, "contextmenu", preventDefault);
+            }
+
             this._releaseHandler(e);
         };
 
         this._pointerdown = (e) => {
             if (e.isPrimary && e.button === 0) {
-                bind(this.document, "pointermove", this._pointermove);
-                bind(this.document, "pointerup", this._pointerup);
-                bind(this.document, "pointercancel", this._pointerup);
-                bind(this.document, "contextmenu", preventDefault);
 
-                this._pressHandler(e);
+                if (!this._clickMoveClick) {
+                    bind(this.document, "pointerup", this._pointerup);
+                    bind(this.document, "pointercancel", this._pointerup);
+                    bind(this.document, "contextmenu", preventDefault);
+                    bind(this.document, "pointermove", this._pointermove);
+                    this._pressHandler(e);
+                } else {
+                    if (this._isDragging) {
+                        bind(this.document, "pointerup", this._pointerup);
+                        bind(this.document, "pointercancel", this._pointerup);
+                    } else {
+                        bind(this.document, "pointermove", this._pointermove);
+                        bind(this.document, "keydown", this._keydown);
+                        this._pressHandler(e);
```

Comment:
It seems to me that it is easy for us to attach more than 1 event handler here for the same event as we have some logic for binding and unbinding events. 

As we do not want to have more than 1 handler - what about unbinding the event in the `bind` function and re-attaching it? This should save us from potential duplications - it would have been nice if we can check if the event has been attached already, but this will require more code.